charts/sourcegraph/templates/syntactic-code-intel/worker.Service.yaml

```diff
@@ -0,0 +1,31 @@
+{{- if .Values.syntacticCodeIntel.enabled -}}
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    prometheus.io/port: "6060"
+    sourcegraph.prometheus/scrape: "true"
+    {{- if .Values.syntacticCodeIntel.serviceAnnotations }}
+    {{- toYaml .Values.syntacticCodeIntel.serviceAnnotations | nindent 4 }}
+    {{- end }}
+  labels:
+    app: syntactic-code-intel-worker
+    deploy: sourcegraph
+    app.kubernetes.io/component: syntactic-code-intel
+    {{- if .Values.syntacticCodeIntel.serviceLabels }}
+      {{- toYaml .Values.syntacticCodeIntel.serviceLabels | nindent 4 }}
+    {{- end }}
+  name: syntactic-code-intel-worker
+spec:
+  ports:
+  - name: http
+    port: {{ .Values.syntacticCodeIntel.properties.workerPort }}
+    targetPort: http
```

Comment:
at a glance https://github.com/sourcegraph/sourcegraph/tree/main/cmd/syntactic-code-intel-worker/shared, it seems like it's pull-based, and doesn't expose any endpoint.

Reply:
Well just like the precise codeintel worker, we expose a general health endpoint: https://github.com/sourcegraph/sourcegraph/blob/main/cmd/syntactic-code-intel-worker/shared/shared.go#L69-L76